### PR TITLE
Linkerd zipkin plugin release 1.5.0

### PR DESCRIPTION
**Pull request by dadjeibaah**
Signed-off-by: Dennis Adjei-Baah <dennis@buoyant.io>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM buoyantio/linkerd:1.4.3
+FROM buoyantio/linkerd:1.5.0
 
 RUN mkdir -p $L5D_HOME/plugins
 COPY plugins/*.jar $L5D_HOME/plugins
```
Comment (klingerf) [inline Dockerfile]: I have a newer version of the docker CLI installed, and this COPY command fails for me, with:

```
Step 3/3 : COPY plugins/*.jar $L5D_HOME/plugins
When using COPY with more than one source file, the destination must be a directory and end with a /
```

Would you mind updating it while you're in here? This works:


```diff
diff --git a/Dockerfile b/Dockerfile
index c220601..7315180 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
 FROM buoyantio/linkerd:1.5.0
 
 RUN mkdir -p $L5D_HOME/plugins
-COPY plugins/*.jar $L5D_HOME/plugins
+COPY plugins/*.jar $L5D_HOME/plugins/
```

---

```
$ docker version
Client:
 Version:           18.06.1-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        e68fc7a
 Built:             Tue Aug 21 17:21:31 2018
 OS/Arch:           darwin/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.06.1-ce
  API version:      1.38 (minimum version 1.12)
  Go version:       go1.10.3
  Git commit:       e68fc7a
  Built:            Tue Aug 21 17:29:02 2018
  OS/Arch:          linux/amd64
  Experimental:     true
 Kubernetes:
  Version:          v1.10.3
  StackAPI:         v1beta2
```

**Review by klingerf — APPROVED** [review]
⭐️ Looks good -- just had one unrelated change to suggest.

File: Dockerfile (hunk 1 above)
Comment (dadjeibaah) [reply Dockerfile]: Thanks for the diff! I wasn't aware of that change. Should have run a docker build to test.

**Comment by klingerf** [comment]
Ah, actually, I just saw this in the sbt logs for this branch:

```
[warn] Found version conflict(s) in library dependencies; some are suspected to be binary incompatible:
[warn] 
[warn] 	* com.twitter:finagle-zipkin-core_2.12:18.9.1 is selected over 18.4.0
[warn] 	    +- io.buoyant:linkerd-zipkin_2.12:1.5.0               (depends on 18.4.0)
[warn] 	    +- io.zipkin.finagle:zipkin-finagle_2.12:1.2.4        (depends on 18.4.0)
[warn] 	    +- io.zipkin.finagle:zipkin-finagle-http_2.12:1.2.4   (depends on 18.4.0)
[warn] 	    +- io.zipkin.finagle:zipkin-finagle-kafka_2.12:1.2.4  (depends on 18.4.0)
[warn] 
[warn] 	* com.twitter:finagle-http_2.12:18.9.1 is selected over 18.4.0
[warn] 	    +- com.twitter:finagle-stats_2.12:18.9.1 ()           (depends on 18.9.1)
[warn] 	    +- io.zipkin.finagle:zipkin-finagle-http_2.12:1.2.4   (depends on 18.4.0)
[warn] 
[warn] 	* com.google.code.findbugs:jsr305:2.0.1 is selected over 1.3.9
[warn] 	    +- com.twitter:util-stats_2.12:18.9.1 ()              (depends on 2.0.1)
[warn] 	    +- com.twitter:util-cache_2.12:18.9.1 ()              (depends on 2.0.1)
[warn] 	    +- com.twitter:finagle-core_2.12:18.9.1 ()            (depends on 2.0.1)
[warn] 	    +- com.google.guava:guava:23.0                        (depends on 1.3.9)
[warn] 
[warn] Run 'evicted' to see detailed eviction warnings
```

I think you need to bump this repo's dependency on [zipkin-finagle](https://github.com/openzipkin/zipkin-finagle) to 2.0.5, which includes the finagle 18.9.1 release. But... I don't see artifacts for that release -- I think we'd need to check with the zipkin folks to see where the 2.x artifacts are being published.

**Comment by dadjeibaah** [comment]
It looks like the bundles are published under `io.zipkin.finagle2`. I will try and add that to see if that breaks linkerd-zipkin 🤞

**Comment by codefromthecrypt** [comment]
let me know if something is yet unpublished...

**Comment by dadjeibaah** [comment]
@adriancole, thanks for checking it. It looks like the latest maven artifact for `io.zipkin.finagle2` is 2.0.4 and has Finagle 18.9.0 as a dependency. We currently have our latest Linkerd 1.5.0 depending on Finagle 18.9.1. Adding the Linkerd dependency to the project gives us a "dependency mismatch" type error. I think the maven artifacts for version 2.0.5 need to be published so that we can add that to linkerd-zipkin's build dependency.

**Comment by codefromthecrypt** [comment]
maybe the index wasn't refreshed when you last looked.. I just checked and
seems 2.0.5 is out there

**Comment by dadjeibaah** [comment]
Thanks @adriancole was able to get things working. Must have been a cache issue 😃.

**Pull request merged**